test/test_helper.rb

```diff
@@ -1,7 +1,6 @@
-ENV["RAILS_ENV"] ||= "test"
-
-require File.expand_path("../../config/environment", __FILE__)
-require "rails/test_help"
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.